app/src/main/java/com/client/fire_and_water/FirstFragment.kt

```diff
@@ -37,29 +25,32 @@ class FirstFragment : Fragment() {
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
-        _binding = FragmentFirstBinding.inflate(inflater, container, false)
+    ): View {
+        _binding = FragmentStartBinding.inflate(inflater, container, false)
         return binding.root
     }
 
-    @RequiresApi(Build.VERSION_CODES.O)
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         mySnackbar = Snackbar.make(view, "can not connect to server", 10)
 
-        binding.buttonFirst.setOnClickListener {
-            val network : Network = (activity as MainActivity).network;
+        binding.startCheckInButton?.setOnClickListener {
+            findNavController().navigate(R.id.action_FirstFragment_to_FifthFragment)
+        }
+
+        binding.startSignInButton.setOnClickListener {
+            val network : Network = (activity as MainActivity).network
             Thread {
                 try {
```

Comment:
Thread?
Isn't it possible to do it here through coroutines?

Reply:
outdated